package.json

```diff
@@ -48,19 +48,19 @@
     "randomcolor": "^0.5.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.7",
-    "@babel/plugin-proposal-class-properties": "^7.7.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.7.6",
-    "@babel/plugin-transform-typescript": "^7.7.4",
-    "@babel/preset-env": "^7.7.7",
-    "@babel/preset-typescript": "^7.7.7",
-    "@babel/runtime": "^7.7.7",
+    "@babel/core": "7.7.7",
```

Comment:
locking down versions in this file. as the local dev server was having issue with moving versions in babel toolings